### PR TITLE
DevOps Website translated.

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

File: content/zh/devops/_index.md [hunk 1]
```diff
@@ -5,47 +5,47 @@ layout: "scenario"
...
+        - content: <span>Source to Image</span> 从源代码构建可再现容器镜像，无需编写 Dockerfile
+        - content: <span>Binary-to-image</span> 将您的制品自动构建成可运行镜像
+        - content: <span>支持自动化构建和推送</span>镜像至任意仓库，并最终部署至 Kubernetes
+        - content: <span>提供卓越的可恢复性和灵活性，</span>您可以在需要补丁时重新构建并重新运行 S2I/B2I
 
-    - title: Use GitOps to implement DevOps, not just culture
+    - title: 使用 GitOps 实现 DevOps（包括其文化）
       image: /images/devops/Clear-insight.png
       contentList:
-        - content: <span>Combine Git with Kubernetes’ convergence, and automates the cloud native Apps delivery</span>
```
Comment (Sherlock113) [inline content/zh/devops/_index.md:37]: Change the English source text to "Combine Git with Kubernetes, automating cloud-native app delivery"

File: content/zh/devops/_index.md [hunk 2]
```diff
@@ -5,47 +5,47 @@ layout: "scenario"
...
-        - content: <span>Designed for teams, offer built-in multitenancy in DevOps project</span>
```
Comment (Sherlock113) [inline content/zh/devops/_index.md:38]: Change the English source text to "Designed for DevOps teamwork on the basis of the multi-tenant system of KubeSphere"

File: content/zh/devops/_index.md [hunk 3]
```diff
@@ -5,47 +5,47 @@ layout: "scenario"
...
-        - content: <span>Liable to be observable,</span> provide dynamic logs for the S2I / B2I build and pipeline
-        - content: Provide audit, alert and notification in pipeline, ensuring issues can be quickly located and solved
-        - content: Support adding Git SCM webhooks to trigger a Jenkins build when new commits are submitted to the branch
+        - content: <span>融合 Git 和 Kubernetes，实现云原生应用自动化交付</span>
+        - content: <span>为团队而设计，在 DevOps 工程中提供内置多租户体系</span>
```
Comment (Sherlock113) [inline content/zh/devops/_index.md:38]: 为团队而设计，在 DevOps 工程中提供内置多租户体系
->
基于 KubeSphere 多租户体系为 DevOps 工程团队合作而打造

File: content/zh/devops/_index.md [hunk 4]
```diff
@@ -5,47 +5,47 @@ layout: "scenario"
...
-        - content: <span>Binary-to-image</span> is the bridge between your artifact and a runnable image
-        - content: <span>Support automatically building and pushing</span> images to any registry, and finally deploy them to Kubernetes
-        - content: <span>Provide excellent recoverability and flexibility</span> as you can rebuild and rerun S2I / B2I whenever a patch is needed
+        - content: <span>Source to Image</span> 从源代码构建可再现容器镜像，无需编写 Dockerfile
+        - content: <span>Binary-to-image</span> 将您的制品自动构建成可运行镜像
+        - content: <span>支持自动化构建和推送</span>镜像至任意仓库，并最终部署至 Kubernetes
+        - content: <span>提供卓越的可恢复性和灵活性，</span>您可以在需要补丁时重新构建并重新运行 S2I/B2I
 
-    - title: Use GitOps to implement DevOps, not just culture
+    - title: 使用 GitOps 实现 DevOps（包括其文化）
```
Comment (Sherlock113) [inline content/zh/devops/_index.md:34]: Change the English source text to "Use GitOps to Implement DevOps"

使用 GitOps 实现 DevOps
->
Use GitOps to Implement DevOps

**Review by Sherlock113 — COMMENTED** [review]
Have a look at my comments and make changes. Thanks.

File: content/zh/devops/_index.md [hunk 5]
```diff
@@ -5,47 +5,47 @@ layout: "scenario"
 css: "scss/scenario.scss"
 
 section1:
-  title: KubeSphere DevOps offers end-to-end workflow and integrates popular CI/CD tools to boost delivery.
```
Comment (Sherlock113) [inline content/zh/devops/_index.md:8]: Change the English source text from "KubeSphere DevOps offers end-to-end workflow" to "KubeSphere DevOps offers end-to-end workflows"

File: content/zh/devops/_index.md (hunk 3 above)
Comment (Felixnoo) [reply content/zh/devops/_index.md:38]: How about one step further:
基于 KubeSphere 多租户体系为 DevOps 工程团队合作而打造
->
基于 KubeSphere 多租户体系，为 DevOps 工程团队打造合作平台

File: content/zh/devops/_index.md (hunk 4 above)
Comment (Felixnoo) [reply content/zh/devops/_index.md:34]: Just to confirm with you that I need to change the ZH sentence "使用 GitOps 实现 DevOps" to the source EN sentence "Use GitOps to Implement DevOps"?

Comment (Sherlock113) [reply content/zh/devops/_index.md:34]: comment updated

File: content/zh/devops/_index.md (hunk 3 above)
Comment (Sherlock113) [reply content/zh/devops/_index.md:38]: sure, that's better

**Pull request merged**